backend/corpora/dataset_processing/process.py

```diff
@@ -226,8 +226,6 @@ def main():
         os.environ["DATASET_ID"],
         os.environ["DROPBOX_URL"],
         "local.h5ad",
-        os.environ["ARTIFACT_BUCKET"],
-        os.environ["CELLXGENE_BUCKET"],
```

Comment:
These lines are added by #799. the called function `download_from_dropbox_url` is only taking 3 arguments resulting in a failure and also not using these parameters. for now I have removed these.